src/libraries/System.Private.CoreLib/src/System/Threading/ThreadPool.cs

```diff
@@ -954,6 +934,33 @@ internal static void PerformWaitOrTimerCallback(_ThreadPoolWaitOrTimerCallback h
 
     public static partial class ThreadPool
     {
+        internal static readonly ThreadPoolWorkQueue s_workQueue = new ThreadPoolWorkQueue();
+        internal static readonly bool s_enableWorkerTracking = GetEnableWorkerTracking();
+
+        /// <summary>Shim used to invoke <see cref="IAsyncStateMachineBox.MoveNext"/> of the supplied <see cref="IAsyncStateMachineBox"/>.</summary>
+        internal static readonly Action<object?> s_invokeAsyncStateMachineBox = new Action<object?>(Invoke.Instance.AsyncStateMachineBox);
```

Comment:
Trying to sneak this in? ;-)  Can we please keep such changes isolated?

Reply:
``ThreadPool+<>C::<.ctor>b__4_0(object)`` bothers me as a method name :)

Moved back to lambda